docs/open-source/graph_memory/overview.mdx

```diff
@@ -232,6 +232,54 @@ m = Memory.from_config(config_dict=config)
 ```
 </CodeGroup>
 
+### Initialize Neptune Analytics
+
+Mem0 now supports Amazon Neptune Analytics as a graph store provider. This integration allows you to use Neptune Analytics for storing and querying graph-based memories.
+
+#### Setup
+
+1. Create a Neptune Analytics instance in your AWS account following the [AWS documentation](https://docs.aws.amazon.com/neptune-analytics/latest/userguide/get-started.html).
+2. Configure your AWS credentials with access to your Neptune Analytics resources using [Configuration and credentials precedence](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-configure.html#configure-precedence).
+
+***Note***: The Neptune Analytics instance must be created using the same vector dimension as the vectors the embedding model creates. See: https://docs.aws.amazon.com/neptune-analytics/latest/userguide/vector-index.html
+
+For example, set the following environment variables:
+<CodeGroup>
+```bash
+export AWS_ACCESS_KEY_ID=your-access-key
+export AWS_SECRET_ACCESS_KEY=your-secret-key
```

Comment:
@acarbonetto, 

1. let's add an example for AWS Session Token. Let's ensure session tokens are supported and tested, and we have examples. 
2. Credentials might be configured in the profile file; let's ensure the profile name can be passed as an alternative option.
3. If the code is running in an environment where an AWS Role is provided, let's make sure that no additional configuration is required (perhaps just validation and examples are sufficient).